libcxx/test/libcxx/diagnostics/limits.nodiscard_extensions.compile.pass.cpp

```diff
@@ -0,0 +1,71 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17
```

Comment:
This should be tested in all versions that it works in, and I don't think there is anything in this test that would require any higher C++ version. Same for the other test.

Reply:
I think this is marked as unsupported, because `[[nodiscard]]` itself is only supported since C++17: https://en.cppreference.com/w/cpp/language/attributes/nodiscard